tests/tests/test-precompile-staking.ts

```diff
@@ -139,8 +139,8 @@ describeDevMoonbeam("Staking - Candidate bond more", (context) => {
       [numberToHex(Number(MIN_GLMR_STAKING))]
     );
 
-    const receipt = await context.web3.eth.getTransactionReceipt(block.txResults[0].result);
-    expect(receipt.status).to.equal(true);
+    // const receipt = await context.web3.eth.getTransactionReceipt(block.txResults[0].result);
+    // expect(receipt.status).to.equal(true);
```

Comment:
@joelamouche This test passes when I comment out these lines, but it fails when I leave them in. I notice that these lines don't appear in the previous test, which is otherwise almost identical. So a few questions for you:

1. Should we actually be checking for the receipt? If so, should we do it in the previous test?
2. Any ideas why we can't get the receipt here?

Reply:
Oh, actually, this seems to be a very new test. Maybe there are EVM fixes that I need to bring in?

Reply:
cc @tgmichel @nanocryk

Reply:
How is it failing for you? I got them passing locally.

Reply:
I restored it, please unresolve if I'm missing something.